### PR TITLE
shorten `start/start-git*` to `start/git*`

### PR DESCRIPTION
**Pull request by casperdcl**
Shorten `/doc/start/start-git*` to `/doc/start/git*`

### CODE REVIEW

**Comment by shcheklein** [comment]
Hey, I would a bit util the issue in the https://github.com/iterative/cml-website/pull/63 is resolved (not merged, but the existing issue with the redirects mechanism itself resolved).

File: redirects-list.json [hunk 1]
```diff
@@ -1,5 +1,11 @@
 [
-  "^https://www\\.cml\\.dev(.*)?                                                          https://cml.dev$1",
-  "^/(?:docs|documentation)(/.*)?$                                                        /doc$1",
-  "^/(.+)/$                                                                               /$1"
+  "^https://(?:www\\.)?cml\\.dev(.*)?                                      https://cml.dev$1",
```
Comment (shcheklein) [inline redirects-list.json:2]: why do we change this record?

File: redirects-list.json [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+  "^/(?:docs|documentation)(/.*)?$                                         /doc$1",
+
+  "^/doc/start/start-github(/.*)?$                                         /doc/start/github",
+  "^/doc/start/start-gitlab(/.*)?$                                         /doc/start/gitlab",
+
+  "^/doc/install(/.*)?$                                                    /doc/install-with-npm 303",
```
Comment (shcheklein) [inline redirects-list.json:8]: why 303 in this case specifically? we are going to change this soon?

Comment (casperdcl) [reply redirects-list.json:8]: yes

File: redirects-list.json (hunk 1 above)
Comment (casperdcl) [reply redirects-list.json:2]: surely we want suppression of `www` just like https://github.com/iterative/dvc.org/blob/88bc65300c3372df0b32ae826869e58ffbf3ef28/redirects-list.json#L2 & https://github.com/iterative/dvc.org/blob/88bc65300c3372df0b32ae826869e58ffbf3ef28/redirects-list.json#L15

Comment (casperdcl) [reply redirects-list.json:2]: oh wait nvm. Not really necessary. Effectively identical.

```suggestion
  "^https://www\\.cml\\.dev(.*)?                                           https://cml.dev$1",
```

**Review by shcheklein — APPROVED** [review]

**Pull request merged**